frontend/src/app/sessions/user-sessions-wrapper/create-sessions/create-persistent-session/create-persistent-session.stories.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { Meta, moduleMetadata, StoryObj } from '@storybook/angular';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { mockTool } from '../../../../../storybook/tool';
+import { PublicLicenseServerWithUsage, Tool } from '../../../../openapi';
+import { ToolWrapperService } from '../../../../settings/core/tools-settings/tool.service';
+import { LicenseUsageWrapperService } from '../../../license-indicator/license-usage.service';
+import { CreatePersistentSessionComponent } from './create-persistent-session.component';
+
+const meta: Meta<CreatePersistentSessionComponent> = {
+  title: 'Session Components/Create Persistent Session',
+  component: CreatePersistentSessionComponent,
```

Comment:
```suggestion
  component: CreatePersistentSessionComponent,
  decorators: [
    componentWrapperDecorator(
      (story) =>
        `<div class="w-[360px] sm:w-[450px]">
          ${story}
        </div>`,
    ),
  ],
```

Make it look realistic in the Story. The values are derived from the sessions component.